polyglot/piranha/demo/match_only/tsx/configurations/rules.toml

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2022 Uber Technologies, Inc.
+#
+# <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+# except in compliance with the License. You may obtain a copy of the License at
+# <p>http://www.apache.org/licenses/LICENSE-2.0
+#
+# <p>Unless required by applicable law or agreed to in writing, software distributed under the
+# License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+[[rules]]
+name = "find_jsx_elements"
+query = """
+(
+    (jsx_element) @jsx_element
+)
+"""
+
+[[rules]]
+name = "find_props_identifiers_within_b_jsx_elements"
+query = """
+(
+    ((identifier) @identifier)
+    (#eq? @identifier "props")
+)
+"""
+[[rules.constraints]]
+matcher = """
+(
+    (jsx_element
+        open_tag: (jsx_opening_element
+            name: (identifier) @name
+            (#eq? @name "b")
+        )
+    ) @j
+)
+"""
+queries = []
+
+[[rules]]
+name = "find_props_identifiers_within_variable_declarators_without_divs"
```

Comment:
Similar to the other comment: `find_props_identifiers_within_variable_declarators_not_within_divs `